src/components/layout/Footer/Footer.tsx

```diff
@@ -1,12 +1,62 @@
+import { QuestionCircleOutlined } from '@ant-design/icons';
+import { Link } from 'react-router-dom';
+
+import { BrandLogo, SunIcon } from '../../icons';
+
 function Footer() {
   return (
-    <div className="flex-row flex-center">
-      <div className="text faded center">
-        v{process.env.npm_package_version}-
-        {process.env.VITE_GITHUB_HASH?.slice(0, 6)}
-        <br />
-        <br />
-        Copyright 2021 - {new Date().getFullYear()}
+    <div
+      className="flex-row flex-space-between"
+      style={{
+        borderTop: '1px solid #292A2B',
+        padding: '30px 100px',
+        boxSizing: 'border-box',
+      }}
+    >
+      <div className="flex flex-row flex-left">
+        <BrandLogo width={'30px'} height={'30px'} fill={'var(--text-faded)'} />
+        <span className="text faded center">
+          Copyright &copy; 2023 ArNS, All rights reserved
+        </span>
+        <Link
+          className="faded text"
+          to={'https://ar.io/arns'}
+          rel="noreferrer"
+          target={'_blank'}
+          style={{ textDecoration: 'underline' }}
+        >
+          Terms & Conditions
+        </Link>
+        <Link
+          className="faded text"
+          to={'https://ar.io/arns'}
+          rel="noreferrer"
+          target={'_blank'}
+          style={{ textDecoration: 'underline' }}
+        >
+          Privacy Policy
+        </Link>
+      </div>
+
+      <span
+        className="flex flex-row flex-right text faded center"
+        style={{ width: 'fit-content', wordBreak: 'keep-all' }}
+      >
+        v{process.env.npm_package_version}
```

Comment:
```suggestion
 v{process.env.npm_package_version}-{process.env.VITE_GITHUB_HASH?.slice(0, 6)}
```

Reply:
added in https://github.com/ar-io/pdns-react/pull/223/commits/2129bc44d57869dda316520cf899a82e05614728